python/marvin/tools/quantities/map.py

```diff
@@ -512,11 +512,15 @@ def inst_sigma_correction(self):
         Correct observed stellar or emission line velocity dispersion
         for instrumental broadening.
         """
+
         if self.datamodel.name == 'stellar_sigma':
 
-            if self._datamodel.parent.release == 'MPL-4':
+            if 'MPL-4' in self._datamodel.parent.aliases:
                 raise marvin.core.exceptions.MarvinError(
                     'Instrumental broadening correction not implemented for MPL-4.')
+            elif 'MPL-6' in self._datamodel.parent.aliases:
+                raise marvin.core.exceptions.MarvinError(
+                    'The stellar sigma corrections in MPL-6 are wrong. Please upgrade to MPL-7.')
```

Comment:
Being PC. I'd say "unreliable" instead of "wrong" and "Please use MPL-7" instead of "upgrade".

Reply:
Easy enough.  It's less aggressive.